From 87a863bd6f21eb01f34fd3a523f4d050a4b235e5 Mon Sep 17 00:00:00 2001
From: Bo Shen <voice.shen@atmel.com>
Date: Thu, 15 Jan 2015 11:07:41 +0800
Subject: [PATCH] HW: info: add missing included file

Include hardware.h to make sure the hardware information can
be written to GPBR. Or else, the AT91C_BASE_GPBR alway not
defined then cause the hardware information can not be passed
through GPBR.

Signed-off-by: Bo Shen <voice.shen@atmel.com>
---
 driver/board_hw_info.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/driver/board_hw_info.c b/driver/board_hw_info.c
index bc27b26..c7a007b 100644
--- a/driver/board_hw_info.c
+++ b/driver/board_hw_info.c
@@ -26,6 +26,7 @@
  * EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 #include "common.h"
+#include "hardware.h"
 #include "string.h"
 #include "board_hw_info.h"
 #include "ds24xx.h"
-- 
1.9.1

